### PR TITLE
[lte]Changes to fix code that is build error prone

### PR DESCRIPTION
**Pull request by ranj8work**
Signed-off-by: Ranjini Iyer <ranj8work@gmail.com>

## Summary
Fixed code which could potentially cause build failures on master

## Test Plan
Testing link is pasted below:

https://app.circleci.com/pipelines/github/magma/magma?branch=build_errors_fixes

## Additional Information

- [NO ] This change is backwards-breaking

### CODE REVIEW

File: circleci/fabfile.py [hunk 1]
```diff
@@ -218,6 +218,12 @@ def _run_remote_lte_integ_test(repo: str, magma_root: str):
         # Copy to the directory CircleCI expects
         local('sudo mkdir -p /tmp/test-results/')
         local('sudo mv test-results/* /tmp/test-results/')
```
Comment (themarwhal) [inline circleci/fabfile.py:220]: Do you mean to replace this line with the new one?

Comment (ranj8work) [reply circleci/fabfile.py:220]: Yes @themarwhal. I thought @tmdzk was reviewing it but if you have questions, I am happy to answer it for you. Thank you for taking a look.

File: circleci/fabfile.py [hunk 2]
```diff
@@ -218,6 +218,12 @@ def _run_remote_lte_integ_test(repo: str, magma_root: str):
...
+        local('if [ "$(ls -A test-results)" ]; then \n'
```
Comment (tmdzk) [inline circleci/fabfile.py:221]: Let's do the logic in python.

**Review by tmdzk — CHANGES_REQUESTED** [review]
The root cause of the CI failure is deeper than that but that's a good catch let's make those changes. (if the folder is empty it means tests were not triggered properly and CI job failed to run)

**Review by tmdzk — CHANGES_REQUESTED** [review]
see my comment about making the logic in python.

**Comment by ekowtaylor** [comment]
@ranj8work , @tmdzk what are the plans to close this?

**Pull request closed without merging**